lib/modem_info/modem_info.c

```diff
@@ -720,6 +720,30 @@ int modem_info_rsrp_register(rsrp_cb_t cb)
 	return 0;
 }
 
+int modem_info_get_fw_uuid(char *buf)
+{
+	int ret;
+
+	ret = nrf_modem_at_scanf("AT%XMODEMUUID", "%*[^:]: %s", buf);
+	return MIN(0, ret);
```

Comment:
The return values will be from the `nrf_modem_` namespace. I think we could instead log the error with debug level and return some other error code that can be documented.